### PR TITLE
Updating docs - Changelog

### PR DESCRIPTION
**Pull request by springcoil**
Ready to merge :)

### CODE REVIEW

**Review by Sjhunt93 — COMMENTED** [review]
LGTM 

(so your sound sounds perfect) -> is wrong in this context, as the preset we added is for very small and loq qaulity files. I would say something like:

' for making sure your audio is the correct format for a target audience'

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+* Friday 13th May 2022
+
+`Users needs are our strategy and our tactics` 
+
+## Features
+We added new endFormat (so your sound sounds perfect) this week we added Alexa preset 
+`code m = apiaudio.Mastering().create(scriptId=GLOBAL_SCRIPT_ID, endFormat="mp3_alexa")`
+
+## Enhancements
+* Improved pronunctuation of our custom voices. We had some pronunctuation issues reported by customers, this adds more natural sounding pauses to our voices.
```
Comment (martinezpl) [inline CHANGELOG.md:12]: pronunciation?

Comment (springcoil) [reply CHANGELOG.md:12]: Yes - the work with adding gaps to speech in the voices (for our own custom voices).

Comment (springcoil) [reply CHANGELOG.md:12]: Ahh yes

**Review by guillem-211 — APPROVED** [review]
looks good to me

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+`Users needs are our strategy and our tactics` 
+v0.6.0
+## Features
+* We added new endFormat (for making sure your audio is the correct format for a target audience) this week we added Alexa preset 
+`code m = apiaudio.Mastering().create(scriptId=GLOBAL_SCRIPT_ID, endFormat="mp3_alexa")`
+* As a developer user I want to share my audio with my team/bosses/business person so they can try and test the wonders of api.audio. We call this *virality link* and it's available in the console now. 
+
+## Enterprise features
+* For our enterprise users we've enabled `sandboxing` this allows you to test safely API requests without using up credits. Please contact your account manager for further details.
```
Comment (GetOn4) [inline CHANGELOG.md:15]: Well, it's production without billing.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## Features
+* We added new endFormat (for making sure your audio is the correct format for a target audience) this week we added Alexa preset 
+`code m = apiaudio.Mastering().create(scriptId=GLOBAL_SCRIPT_ID, endFormat="mp3_alexa")`
+* As a developer user I want to share my audio with my team/bosses/business person so they can try and test the wonders of api.audio. We call this *virality link* and it's available in the console now. 
+
+## Enterprise features
+* For our enterprise users we've enabled `sandboxing` this allows you to test safely API requests without using up credits. Please contact your account manager for further details. 
+
+## Enhancements
+* Improved pronunciation of our custom voices. We had some pronunctuation issues reported by customers, this adds more natural sounding pauses to our voices.
```
Comment (GetOn4) [inline CHANGELOG.md:18]: this is not in production.

**Review by GetOn4 — APPROVED** [review]
LGTM - See my comments.

File: CHANGELOG.md (hunk 2 above)
Comment (springcoil) [reply CHANGELOG.md:15]: Yeah but we'll add more stuff to it as well.

**Pull request merged**